protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -258,6 +301,126 @@ open class CodeGenerator(val file: File, val kotlinTypeMappings: Map<String, Str
         }.line("}")
     }
 
+    protected fun writeMessageToJsonMapperExtension(type: File.Type.Message, fullTypeName: String) {
+        line().line("fun $fullTypeName.toJsonMapper(): $fullTypeName.JsonMapper =").indented {
+            line("$fullTypeName.JsonMapper(").indented {
+                val allFields = type.sortedStandardFieldsWithOneOfs()
+                allFields.forEachIndexed { i, (field, _) ->
+                    when {
+                        (field.type == File.Field.Type.ENUM) -> {
```

Comment:
🔧 Change to `when (field.type) { ... }`

Reply:
Must've had more cases in there at some point... changed.